### PR TITLE
add load_containers()

### PR DESCRIPTION
**Pull request by zackarno**
You will recognize some of this code from signals. 

I had originally utilized even more of the code from signals in the `read_az_file_v1` branch which mainly includes a variation of the signals function to download a tempfile and read it in. I was using this approach originally, but in more recent work I've started just using the  two functions in this branch. I have just been copy pasting them from project to project so figured it's about time to put them here.

I feel like this approach will eventually be cleaner and require less env vars. I think  we should eventually build in `read` style functions plus alot more, but these seem like some good lower level functions to start with and build off

### CODE REVIEW

File: R/load_containers.R [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+#'
+#' # You can also list as many containers as you want.
+#' pc <- load_containers(containers = c("global", "projects"))
+#' AzureStor::list_blobs(
+#'   container = pc$GLOBAL_CONT,
+#'   dir = "raster/cogs"
+#' )
+#' @export
+load_containers <- function(
+    containers = c("global", "projects"),
```
Comment (zackarno) [inline R/load_containers.R]: would be kind of cool if we could use:

```
AzureStor::list_blob_containers()
```

in the event that no containers are specified as well as for checking container args. However it seems somehow our blob permissions do not allow this. I guess if we specify a container that doesn't exist we will get the appropriate warning anyways from `{AzureStor}` so not really critical for checking args

**Comment by caldwellst** [comment]
Please stop force pushing! You've erased my draft review haha.

**Comment by zackarno** [comment]
> Please stop force pushing! You've erased my draft review haha.

oh crap! didn't realize that was possible - my bad!

Sometimes I commit -> push and realize there was tiny extra needed change. So what I usually do is:

```
git commit --amend --no-edit
```

But after doing that I can't simply `git push`. It makes you `git push -f` 

Is there a better way to do add to the previous commit?

I guess just better to make a new commit if not!

**Comment by caldwellst** [comment]
Ha, usually is okay I think, not sure what happened. But I think just add a new commit especially if you're squashing and merging later, that way you don't have to be too careful about commit history when you make errors like that!

**Comment by zackarno** [comment]
I updated this function to be much nicer and now we can list_containers() with right sas key. Going to merge to main for more use & exposure

**Pull request merged**